frontend/src/app/signUp/Organization/utils.tsx

```diff
@@ -0,0 +1,123 @@
+import * as yup from "yup";
+
+import { TextWithTooltip } from "../../commonComponents/TextWithTooltip";
+import { phoneNumberIsValid } from "../../patients/personSchema";
+import { liveJurisdictions } from "../../../config/constants";
+
+import { OrganizationCreateRequest } from "./OrganizationForm";
+
+export const OrganizationTypeEnum = {
+  airport: "Airport/Transit station",
+  assisted_living: "Assisted living facility",
+  camp: "Camp",
+  university: "College/University",
+  correctional_facility: "Correctional facility",
+  employer: "Employer",
+  fqhc: "Federally Qualified Health Center (FQHC)",
+  government_agency: "Government agency",
+  shelter: "Homeless shelter",
+  hospice: "Hospice",
+  hospital: "Hospital or clinic",
+  k12: "K-12 school",
+  lab: "Lab",
+  nursing_home: "Nursing home",
+  other: "Other",
+  pharmacy: "Pharmacy",
+  primary_care: "Primary care / Mental health outpatient",
+  treatment_center: "Substance abuse treatment center",
+  urgent_care: "Urgent care",
+};
+
+export const organizationFields = [
+  [
+    "name",
+    <span>
+      What's the name of your{" "}
+      <TextWithTooltip
+        text="organization?"
+        tooltip="Organizations have multiple testing facilities or locations as part of their network."
+        position="top"
+      />
+    </span>,
+    true,
+    "If you have only one location, enter that name for both your organization name and testing facility name.",
```

Comment:
**content question:** @Rebecca-LM can you take a look at the copy here? Since the form doesn't include facility information anymore, I'm not sure if the subheader should still refer to facilities.

Reply:
@emmastephenson totally! I think we should remove and included this in my edits for Nick in my comment below.